src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,42 @@
+import { useParams} from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getArticleById } from "../utils/api";
+
+const SingleArticle = () => {
+    const [isLoading, setIsLoading] = useState(true);
+   const {article_id} = useParams()
+   const[article,setArticle] = useState({
+      "article_id": 1,
+      "title": "Running a Node App",
+      "topic": "coding",
+      "author": "jessjelly",
+      "body": "This is part two of a series on how to get up and running with Systemd and Node.js. This part dives deeper into how to successfully run your app with systemd long-term, and how to set it up in a production environment.",
+      "created_at": "2020-11-07T06:03:00.000Z",
+      "votes": 0,
+      "comment_count": 8
+    })
```

Comment:
I'd avoid hardcoding your state here, let the useEffect handle it for you :)

Reply:
oops forgot to take it out